### PR TITLE
Adds an interface option to position the item pickup panel.

### LINKED ISSUE #29415: Option to change the loot table from left to right
Is your feature request related to a problem? Please describe.
While this is not a major problem by a wide margin, it is rather difficult to adjust to the new loot table. Back then every information is relayed on the table to the right. Now with the newest update, whenever you want to check an item on the ground your head need to be jerked to the left to see everything. Ergonomic wise it was a bit uncomfortable. I think the loot table in the old version works really well so far.

Describe the solution you'd like
While i think some might like this feature, i think removing this entirely is not necessary. I would love it if there's simply an option to move the loot table back to the right. the 'classic' mode.

Describe alternatives you've considered
So far the solution above is the only one i have, i don't think it's removing anything. just adding an option for the user interface.

Additional context
Here is the screenshot provided of the current loot table on the left.

### PR DESCRIPTION
**Pull request by Petethegoat**
#### Summary
```SUMMARY: Interface "Added an option to position the item pickup panel."```

#### Describe the solution
Adds a new option, "PICKUP_POSITION", which can be set to left, right, or overlapping.
Position the pickup panel to avoid the sidebar, or to overlap it if that option is in use.

Sidebar and pickup panel set to the right side:
right, avoiding sidebar
Overlapping, with sidebar on the left side:
sidebar set to left, with pickup panel overlapping

#### Additional context
Closes #29415.

eso discussed the issue of settings bloat, which this certainly contributes to in it's present state.
Based on the reactions on the original issue, I'd venture to say it'd be worth merging as is, but I won't be offended if we want to delay this until the overall settings issues have been resolved.
This isn't so complicated that it can't sit in a branch and be easily updated down the line.

### CODE REVIEW

File: src/options.cpp [hunk 1]
```diff
@@ -1458,7 +1463,6 @@ void options_manager::add_options_interface()
 
     add( "MORALE_STYLE", "interface", translate_marker( "Morale style" ),
          translate_marker( "Morale display style in sidebar." ),
-         //~ aim bar style - bars or numbers
```
Comment (Night-Pryanik) [inline src/options.cpp]: Why did you remove this translator comment?

Comment (Petethegoat) [reply src/options.cpp]: Thought it was just a misplaced copy paste- is morale display actually the same as the aiming style option?

Comment (Night-Pryanik) [reply src/options.cpp]: Oh, didn't notice that was a copypaste. Yeah, you're right. You could alter the comment to correctly represent what should be translated, or remove the comment entirely, on your discretion.

**Comment by Petethegoat** [comment]
Can't believe msvc didn't complain about `pickupX`being uninitialized, and I could have sworn I remembered to run astyle. Will fix those tomorrow.

**Comment by paulenka-aleh** [comment]
Thank you a lot @Petethegoat for imlementing it!
